docs/UserGuide.md

```diff
@@ -196,44 +196,34 @@ Example:
 - Marks the status of the applicant with ID 106 as accepted.
 
 
-### Adding a Job: `job add` [coming soon]
-Adds a job attribute
+### Adding a Job: `addjob`
+Adds a new job posting to ReCLIne
 
-Format: `job add [ID*] ed/[EDUCATION] l/[LOCATION] s/[SALARY] sp/[SPECIALISATION] d/[DURATION]`
+Format: `addjob jt/[JOB TITLE] c/[COMPANY] a/[ADDRESS] q/[QUALIFICATION] pos/[POSITION] sal/[SALARY]`
 
 *flag/[Attribute]:*
 
-`[ID*]`: Unique ID for the job instance. The ID must be a positive integer 1, 2, 3..
+`jt/[JOB TITLE]`: Job title of the job
 
-`e/[EDUCATION]`: Education level requirement for the job
+`c/[COMPANY]`: The company for the job opening 
 
-`l/[LOCATION]`: Location of the job
+`a/[ADDRESS]`: Location of the job 
 
-`s/[SALARY]`:  Salary of job, based on how much the job pays a month. The SALARY must be a positive integer
-1000,2000,3000,...
+`q/[QUALIFICATION]`: Education qualification required for the job
 
-`sp/[SPECIALIZATION]`: Field of Specialisation required for the job
+`pos/[POSITION]`: Type of job, whether part-time or full time.
 
-`d/[DURATION]`: Duration of job in months. ie d/8 represents 8 months
+`s/[SALARY]`:  Salary of job, based on how much the job pays a month. The SALARY must be a positive integer
```

Comment:
Should state that this is a range. Something similar to my PR?

Reply:
Ok added it as a note